CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+-   Permissions for service user are only to save scripts, replicate scripts and read user groups [#46](https://github.com/orbinson/aem-groovy-console/issues/46)
```

Comment:
imho it should be market as 'breaking changes'

Reply:
that was the plan